js-html-games/numScript.js

```diff
@@ -0,0 +1,133 @@
+const custom = document.getElementById("custom");
+const random = document.getElementById("random");
+const numForm = document.getElementById("numForm");
+const range = document.getElementById("range");
+const statement = document.getElementById("statement");
+const bttn = document.getElementById("set");
+const hint = document.getElementById("hint");
+let bounds = [];
+let correctNum = 0;
+
+function clear() {
+    bgColor("white");
+    setText("");
+}
+
+function bgColor(color) {
+    document.body.style.backgroundColor = color;
+}
+
+function setText(text) {
+    hint.innerText = text;
+}
+
+function addFormSubmit(corr) {
+    numForm.addEventListener('submit', function (event) {
+        event.preventDefault();
+        const guess = document.getElementById("guess").value;
+        checkNum(guess, corr);
+    })
+}
+
+function hotOrCold(num) {
+    if (num === 0) {
+        bgColor("#8dcc9e");
+        setText("CORRECT! GOOD JOB!");
+    }
+    else if (num > 0 && num <= 10) {
+        bgColor("#e8f781");
+        setText("Close! Keep going! (within 10)");
+    }
+    else if (num > 10 && num <= 100) {
+        bgColor("#f0ce7a");
+        setText("Not quite! (within 100)");
+    }
+    else if (num > 100 && num <= 1000) {
+        bgColor("#d4733b");
+        setText("Not quite! (within 1000)");
+    }
+    else {
+        bgColor("#d14134");
+        setText("Within range, but you're way off!");
+    }
+}
+
+function upDown(num) {
+    if (num === 0) {
+        bgColor("#8dcc9e");
+        setText("CORRECT! GOOD JOB!");
+    }
+    else if (num > 0) {
+        bgColor("#e8f781");
+        setText("Go down!");
+    }
+    else if (num < 0) {
```

Comment:
if num is not zero, and is not greater than zero, then the only available set of values we can access is num < 0. In that case, we can do 
```
if (num === 0){}
else if (num > 0){}
else {}
```

Reply:
recommend using if, else if, else branching if you can for clarity. else if's usually indicate a more specific scenario